invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LocationList/LocationList.js

```diff
@@ -55,7 +60,10 @@ export default class LocationList extends Component {
 }
 
 LocationList.propTypes = {
-  data: PropTypes.array.isRequired,
+  data: PropTypes.oneOfType([
+    PropTypes.object.isRequired,
```

Comment:
why do you allow to have object in the data ?

Reply:
Because error state and data state is different right now :)